### ISSUE TITLE
More quadrilateral stickiness

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/quadrilateral#271]
Vertices and sides are still getting stuck after #41. It seems to happen when moving around the model bounds. Here is an example from @BLFiedler 
![image](https://user-images.githubusercontent.com/6396244/205988882-2f062c9e-6cf3-4680-90bf-04c7c2fc8234.png)

Vertex B was close to the left side and could not move further with keyboard but could move more with a mouse.

**Comment by terracoda** [comment]
Another example. Corner C got stuck here.
<img width="848" alt="image" src="https://user-images.githubusercontent.com/1902116/206476113-073492ab-6178-4633-b6dc-952bcf339dc9.png">

**Comment by jessegreenberg** [comment]
In this trapezoid I am not able to move side AB further right:
![image](https://user-images.githubusercontent.com/6396244/206791138-f7fdd262-6342-4777-b5d8-a6ea41823afb.png)

The function added to fix #180 is reporting that the proposed position for VertexA is outside of its allowed drag area.
```js
shapeAllowed = this.customShapeContainsPoint( testVertex.dragAreaProperty.value!, testVertex.positionProperty.value );
```

**Comment by jessegreenberg** [comment]
Alright, #180 and #284 are closed again. I have been playing with the sim for 5 minutes or so and have not seen any more stickiness. I think this can be closed.
